### ISSUE TITLE
Proposed: Recommend ontologies are server via HTTP

### ISSUE THREAD

**Issue by alanruttenberg** [OBOFoundry/purl.obolibrary.org#361]
A number of ontologies redirect to ftp servers. FTP doesn't play nice with HTTP caches, since it doesn't handle HEAD, or Etags. Not the end of the world, but still enough, I think, to recommend serving over HTTP. 

Currently only 3 ontologies serve over FTP: PRO, ChEBI, and RS (Rat Strain).
I've already requested that PRO switch to HTTP, and will probably make a request that ChEBI do the same. I'm not familiar with RS developers, but there's a contact listed, and I don't mind asking them as well.

**Comment by jamesaoverton** [comment]
I agree that this is a good recommendation. Caches, HEAD, and Etags are especially useful for large ontologies with frequent releases such as PRO and ChEBI.
